frameworks/kafka/tests/test_sanity.py

```diff
@@ -105,7 +97,9 @@ def test_custom_zookeeper():
     # wait for brokers to finish registering
     test_utils.broker_count_check(config.DEFAULT_BROKER_COUNT, service_name=foldered_name)
 
-    zookeeper = sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, "endpoints zookeeper")
+    zookeeper = sdk_networks.wait_for_endpoint_info_string(
+        config.PACKAGE_NAME, foldered_name, "zookeeper"
+    )
     assert zookeeper.rstrip("\n") == zk_path
```

Comment:
It is my understanding that `strip()` would also remove `\n` and as such this `rstrip()` is now redundant.

Reply:
True